docs/customization.md

```diff
@@ -154,6 +154,45 @@ variants:
             PATH: "/cpu/bin:$PATH"
 ```
 
+#### template envs
+
+.. versionadded: 0.31
+
+Template variables for extra environment variables can be configured in the
```

Comment:
So these are only included in the logic for expanding template values, but are not "defaults" for these variables because they aren't passed unless the variant-specific configuration sets the variable, too?

Reply:
That's correct.

Reply:
That seems likely to introduce some confusion. What's the downside to just treating them as global defaults?